docs/pipeline-components-and-applications/loaders-storage-targets/bigquery-loader/previous-versions/bigquery-loader-1.x/index.md

```diff
@@ -0,0 +1,315 @@
+
+<table><tbody><tr><td><strong>Kind</strong></td><td><strong>Populated by</strong></td><td><strong>Consumed by</strong></td><td><strong>Data format</strong></td></tr><tr><td>Input subscription</td><td>Enriched events topic</td><td>Loader / StreamLoader</td><td>canonical <code>TSV + JSON</code> enriched format</td></tr><tr><td>Types topic</td><td>Loader / StreamLoader</td><td>Types subscription</td><td><code>iglu:com.snowplowanalytics.snowplow/shredded_type/jsonschema/1-0-0</code></td></tr><tr><td>Types subscription</td><td>Types topic</td><td>Mutator</td><td><code>iglu:com.snowplowanalytics.snowplow/shredded_type/jsonschema/1-0-0</code></td></tr><tr><td>Bad row topic</td><td>Loader / StreamLoader</td><td><a href="/docs/pipeline-components-and-applications/loaders-storage-targets/google-cloud-storage-loader/">GCS Loader</a></td><td><code>iglu:com.snowplowanalytics.snowplow.badrows/loader_iglu_error/jsonschema/2-0-0</code><br/><code>iglu:com.snowplowanalytics.snowplow.badrows/loader_parsing_error/jsonschema/2-0-0</code><br/><code>iglu:com.snowplowanalytics.snowplow.badrows/loader_runtime_error/jsonschema/1-0-1</code></td></tr><tr><td>Failed insert topic</td><td>Loader / StreamLoader</td><td>Failed insert subscription</td><td><code>BigQuery JSON</code></td></tr><tr><td>Failed insert subscription</td><td>Failed insert topic</td><td>Repeater</td><td><code>BigQuery JSON</code></td></tr></tbody></table>
+
+## Setup guide
+
+### Configuration file
+
+Loader / StreamLoader, Mutator and Repeater accept the same configuration file in HOCON format. An example of a minimal configuration file can look like this:
+
+```json
+{
+  "projectId": "com-acme"
+
+  "loader": {
+    "input": {
+      "subscription": "enriched-sub"
+    }
+
+    "output": {
+      "good": {
+        "datasetId": "snowplow"
+        "tableId": "events"
+      }
+
+      "bad": {
+        "topic": "bad-topic"
+      }
+
+      "types": {
+        "topic": "types-topic"
+      }
+
+      "failedInserts": {
+        "topic": "failed-inserts-topic"
+      }
+    }
+  }
+
+  "mutator": {
+    "input": {
+      "subscription": "types-sub"
+    }
+
+    "output": {
+      "good": ${loader.output.good} # will be automatically inferred
+    }
+  }
+
+  "repeater": {
+    "input": {
+      "subscription": "failed-inserts-sub"
+    }
+
+    "output": {
+      "good": ${loader.output.good} # will be automatically inferred
+
+      "deadLetters": {
+        "bucket": "gs://dead-letter-bucket"
+      }
+    }
+  }
+
+  "monitoring": {} # disabled
+}
+```
+
+The loader takes command line arguments `--config` with a path to the configuration hocon file and `--resolver` with a path to the Iglu resolver file. If you are running the docker image then you should mount the configuration files into the container:
+
+<CodeBlock language="bash">{
+`docker run \\
+    -v /path/to/configs:/configs \\
+    snowplow/snowplow-bigquery-streamloader:1.7.1 \\
```

Comment:
I recommend creating a new variable like `versions.bqLoader1x` in case we release 1.7.2 in the meantime and need to adjust this.